### ISSUE TITLE
duden.de - new ads insertion

### ISSUE THREAD

**Issue by Hrxn** [uBlockOrigin/uAssets#5453]
### URL(s) where the issue occurs

`https://www.duden.de/suchen/dudenonline/beispiel`
`https://www.duden.de/rechtschreibung/Beispiel`

### Describe the issue

Very recent site change, huge banners insertion top-center and at the right.

### Screenshot(s)

not applicable 

### Versions

- Browser/version: Chrome 74.0.3729.108 (latest Stable channel, Win 64)
- uBlock Origin version: 1.18.16

### Settings

All uBO default settings
\+ EasyList DEU

**Comment by Hrxn** [comment]
Hey thanks guys, you are very quick!

I confirm that this is working for me, new insertion is gone and site still functions as before.

I want to refine the filter experience a bit though, can you help me with that?

First, on each search (`duden.de/suchen/dudenonline/<s>`) and and result (`duden.de/rechtschreibung/<r>`) page are two banners directly to the right of the actual content, I realize that they are not 3rd part advertising, but own "suggestions", so this would maybe only qualify as an annoyance.
I tried these simple cosmetic filters for this, seems to work fine:
```
www.duden.de##.scene--style_default.scene
www.duden.de##.essay--compact.essay
```
I don't know if you consider this worth of adding. I personally prefer it this way, for sure.

Also, another thing, pages like those above (and the 'Rechtschreibprüfung' mentioned in the commit discussion, as well as most other pages) waste a lot of screen space with a grey column on the right side, without actually displaying anything there. Unlike the actual homepage itself, which displays just fine.

I tried a style filter like this, can you tell me if that is working for you?
```
www.duden.de##body:style(width: auto !important; padding: 0 !important; margin: 0 !important)
```

**Comment by okiehsch** [comment]
> I realize that they are not 3rd-party advertising, but own "suggestions"

That is self-advertisment, in my opinion, and no filters will be added for such cases.

Your style filter does not work on my end and I can't seem to find a working style filter myself.

Just fyi you can remove the `www.` part of all your filters.

**Comment by okiehsch** [comment]
`duden.de##.tabloid__sheet--has-sidebar:style(width: 118% !important;)`
seems to work for me.

**Comment by Hrxn** [comment]
> Just fyi you can remove the `www.` part of all your filters.

True, although it wouldn't matter in this case, because `https://duden.de/xxx` always redirects to `https://www.duden.de/xxx` . I made sure to test this.

And uBO's element picker creates filters with URLs as in the Browser, so in this case it had the `www.` part.

But you're right, the idiomatic way to write filters is to always omit the `www.` part, apparently. I've changed this for all my filters now, and will keep that in mind for the future.
(On a somewhat related sidenote, I wish this whole `www` thing would finally be purged from the Internet entirely, I hate that this is used so inconsistently.)

> That is self-advertisment, in my opinion, and no filters will be added for such cases.

Fair enough, fine with me.

> Your style filter does not work on my end and I can't seem to find a working style filter myself.

Sorry for the misunderstanding, but yeah, the filter did not work for me either. It's targeting the wrong element (`body`), I was still searching for the right one myself at that moment. I usually start with the HTML source and try to find a way forward from there on, but these modern JS-heavy sites are probably too obfuscated for me. I wish I knew a good general solution to debug this kind of stuff.

> `duden.de##.tabloid__sheet--has-sidebar:style(width: 118% !important;)`

Yes, seems to work for me as well. I think I still see a small gap, but definitely a huge improvement. Although I wonder about this oddly specific value, why wouldn't `width: auto` work as well?
